src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowCreate.cs

```diff
@@ -17,7 +22,7 @@ namespace Microsoft.SqlTools.ServiceLayer.EditData.UpdateManagement
     /// </summary>
     public sealed class RowCreate : RowEditBase
     {
-        private const string InsertStatement = "INSERT INTO {0}({1}) VALUES ({2})";
+        private const string InsertStart = "INSERT INTO {0}({1})";
 
         private readonly CellUpdate[] newCells;
```

Comment:
I've definitely been asked to add _ in a PR before, but I guess if we don't follow that standard it's not an issue.